rexclient/BUILD

```diff
@@ -23,6 +23,6 @@ go_test(
         ":rexclient",
         "///third_party/go/github.com_bazelbuild_remote-apis-sdks//go/pkg/digest",
         "///third_party/go/github.com_bazelbuild_remote-apis//build/bazel/remote/execution/v2",
-        "///third_party/go/github.com_stretchr_testify//assert",
```

Comment:
I think these are better not changed... the build failures seem like the subpackage builds okay but the whole thing doesn't

Reply:
I've fixed the `go.mod` so these work as expected now.